### DISCUSSION TITLE
"Template parsing error, aborting"

### DISCUSSION THREAD

**Discussion by clcoble** [SilentVoid13/Templater#844] [Help]
Hey all,

Not sure if this is the best place to ask this question, but I cannot get Templater to work anymore. Every time I try to run a Templater hot key is gives me the "Templater Error: Template parsing error, aborting. TP is not defined" error in the console. It also says "VM166 plugin:templater-obisidian:82" next to the error if that helps. I've attempted to uninstall/reinstall Templater without luck.

I've also noticed that it seems to do this when I create a new note on the mobile app. Before, I would just recreate a new Vault since I didn't have many notes but now my Vault has a lot of info. in it. Hoping I don't have to restart a new Vault.

Thanks so much.

**Comment by AB1908** [comment]
Any user scripts? You need to share the template as well.

**Reply by clcoble** [reply]
Hi AB1908,

Please see below for the template. "email:" to "category:" is all part of the Front Matter.

`
---
email:
aliases:
date_last_spoken:
follow_up:
category:
---


# [[<% TP.FILE.TITLE %>]]`

**Reply by clcoble** [reply]
I noticed if I change the TP.FILE.TITLE code to lower case it works. I'm fine with that but thought you could make it all caps.

**Reply by AB1908** [reply]
No, unfortunately it's case sensitive. Do you need further assistance?
